tsuru/client/deploy.go

```diff
@@ -645,3 +646,72 @@ func (c *AppDeployRebuild) Run(context *cmd.Context, client *cmd.Client) error {
 	}
 	return nil
 }
+
+type AppDeployRollbackUpdate struct {
+	cmd.GuessingCommand
+	image   string
+	reason  string
+	disable bool
+	fs      *gnuflag.FlagSet
+}
+
+func (c *AppDeployRollbackUpdate) Info() *cmd.Info {
+	desc := `Locks an existing image of an app. You can list images with "tsuru app-deploy-list -a <appName>"
+
+::
+
+    The [-i/--image] flag is the name of an app image.
+
+    The [-d/--disable] flag enables or disables a rollback to an image, by default if not parsed it enables a rollback to an image, otherwise it'll disable it.
```

Comment:
This description is kinda confusing. I'd change to something like `The [-d/--disable] flag disables an image rollback. The default behavior (omitting this flag) is to enable it.`